_includes/community/community-meetings-table.html

```diff
@@ -63,3 +63,10 @@
 </table>
 <br />
 </div>
+
+
+  <div align ="center">
+    <div align="center"><h3>Checkout Our Youtube Playlist</h3></div>  
+     <iframe width=65% height="315" src="https://www.youtube.com/embed/videoseries?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Increase the height a bit. The viewing ratio doesn't look good right now. Other than that great job!